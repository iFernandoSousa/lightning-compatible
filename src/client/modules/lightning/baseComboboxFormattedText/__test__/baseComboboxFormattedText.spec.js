/*
 * Copyright (c) 2019, salesforce.com, inc.
 * All rights reserved.
 * SPDX-License-Identifier: MIT
 * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/MIT
 */

import { createElement } from 'lwc';
import Element from 'c/baseComboboxFormattedText';

const createComponent = (params = {}) => {
    const element = createElement('base-combobox-formatted-text', {
        is: Element
    });

    Object.assign(element, params);
    document.body.appendChild(element);
    return element;
};

describe('base-combobox-formatted-text', () => {
    it('renders', () => {
        const element = createComponent({
            text: [{ text: 'text one' }, { text: 'text two' }]
        });

        expect(element).toMatchSnapshot();
    });
});
