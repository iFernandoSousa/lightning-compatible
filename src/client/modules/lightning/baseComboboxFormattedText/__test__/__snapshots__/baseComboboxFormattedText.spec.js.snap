// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base-combobox-formatted-text renders 1`] = `
<base-combobox-formatted-text>
  #shadow-root(open)
    text one
    text two
</base-combobox-formatted-text>
`;
